libsolidity/interface/CompilerStack.cpp

```diff
@@ -126,14 +127,14 @@ void CompilerStack::setRemappings(vector<Remapping> const& _remappings)
 
 void CompilerStack::setEVMVersion(langutil::EVMVersion _version)
 {
-	if (m_stackState >= ParsingSuccessful)
+	if (m_stackState >= ParsingSuccessful && !m_importedSources)
```

Comment:
Why do you need the second check? If you import, then the state should go to ParsingSuccessful anyway.

Reply:
true. This is a leftover from when i had it arranged differently

Reply:
yes, that was a leftover from back when i had the code arranged differently.